src/pages/saksbehandling/revurdering/formue/RevurderFormueUtils.ts

```diff
@@ -0,0 +1,264 @@
+}
+export interface VerdierFormData {
+    verdiPåBolig: string;
+    verdiPåEiendom: string;
+    verdiPåKjøretøy: string;
+    innskuddsbeløp: string;
+    verdipapir: string;
+    kontanterOver1000: string;
+    stårNoenIGjeldTilDeg: string;
+    depositumskonto: string;
+}
+
+const tomFormue: VerdierFormData = {
+    verdiPåBolig: '0',
+    verdiPåEiendom: '0',
+    verdiPåKjøretøy: '0',
+    innskuddsbeløp: '0',
+    verdipapir: '0',
+    kontanterOver1000: '0',
+    stårNoenIGjeldTilDeg: '0',
+    depositumskonto: '0',
+};
+
+export const leggTilNyPeriode = (epsFnr: Nullable<string>): FormueData => {
+    return {
+        epsFnr: epsFnr,
+        periode: { fraOgMed: null, tilOgMed: null },
+        søkersFormue: tomFormue,
+        epsFormue: epsFnr ? tomFormue : null,
+        begrunnelse: null,
+    };
+};
+
+export const getDefaultValues = (formueVilkår: Nullable<FormueVilkår>, epsFnr: Nullable<string>): FormueFormData => {
+    if (!formueVilkår) {
+        return {
+            formue: [leggTilNyPeriode(epsFnr)],
+        };
+    }
+
+    return {
+        formue: formueVilkår.vurderinger.map((formue) => {
+            return {
+                epsFnr: epsFnr,
+                periode: {
+                    fraOgMed: new Date(formue.periode.fraOgMed),
+                    tilOgMed: new Date(formue.periode.tilOgMed),
+                },
+                søkersFormue: formueGrunnlagVerdierTilVerdierFormData(formue.grunnlag?.søkersFormue) ?? tomFormue,
+                epsFormue: epsFnr
+                    ? formueGrunnlagVerdierTilVerdierFormData(formue.grunnlag?.epsFormue) ?? tomFormue
+                    : null,
+                begrunnelse: formue.grunnlag?.begrunnelse ?? '',
+            };
+        }),
+    };
+};
+
+export const formueFormDataTilFormuegrunnlagRequest = (data: FormueData[]): FormuegrunnlagFormue => {
+    return data.map((formue) => {
+        return {
+            periode: {
+                /* eslint-disable @typescript-eslint/no-non-null-assertion */
+                fraOgMed: DateUtils.toIsoDateOnlyString(formue.periode.fraOgMed!),
+                tilOgMed: DateUtils.toIsoDateOnlyString(formue.periode.tilOgMed!),
+                /* eslint-enable @typescript-eslint/no-non-null-assertion */
+            },
+            //parseVerdier skal ikke kunne returnere null når input-parameteren ikke er null
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            søkersFormue: formDataVerdierToNumber(formue.søkersFormue)!,
+            epsFormue: formDataVerdierToNumber(formue.epsFormue),
+            begrunnelse: formue.begrunnelse,
+        };
+    });
+};
+
+//hvis fraOgMed ikke er utfyllt, eller vi ikke finner en match for fraOgMed,
+//bruker vi den høyeste g-verdien som default
+export const getSenesteGVerdi = (fraOgMed: Nullable<Date>, formuegrenser: Formuegrenser[]) => {
+    const sortert = formuegrenser.slice().sort((a: Formuegrenser, b: Formuegrenser) => {
```

Comment:
Burde vi kanskje håndtere at noen kan finne på å sende inn et tomt array for `formuegrenser`?

Reply:
Hva burde vi gjøre da? 😅 Vise en melding om at vi ikke har en G-verdi for frontend?

Reply:
Hehe, kanskje noe sånt ja 😅  Det er vel en klassisk "skal-ikke-skje-situasjon", men hvis vi legger inn tydeligere håndtering av det så blir det forhåpentligvis enklere å finne ut av dersom det inntreffer.